acceptance/scripts/generic/testrun.sh

```diff
@@ -14,9 +16,58 @@ export BEAKER_HELPER="acceptance/lib/helper.rb"
 
 bundle install --path vendor/bundle
 
-bundle exec genconfig2 $GENCONFIG_LAYOUT > $BEAKER_CONFIG
+#TODO: Someday, when our Rakefile is refactored, this section will go away
+BEAKER="bundle exec beaker --debug"
+BEAKER="$BEAKER --type aio"
+BEAKER="$BEAKER --keyfile $BEAKER_KEYFILE"
+BEAKER="$BEAKER --helper $BEAKER_HELPER"
+BEAKER="$BEAKER --options-file $BEAKER_OPTIONS"
+BEAKER="$BEAKER --post-suite $BEAKER_POSTSUITE"
+BEAKER="$BEAKER --load-path acceptance/lib"
+
+
+case $1 in
+  -p | --p* )
+  bundle exec genconfig2 $GENCONFIG_LAYOUT > $BEAKER_CONFIG
+    
+  if [ -z "$PACKAGE_BUILD_VERSION" ]; 
```

Comment:
Yes, it blows up, but the error message isn't intelligible, and it takes a few minutes to get into the test before you discover that something is wrong.